core/modules/css/CssAccess.cc

```diff
@@ -503,7 +503,8 @@ CssAccess::getPartTableParams(std::string const& dbName,
 
     std::vector<std::string> subKeys{"partitioning", "partitioning/subChunks",
         "partitioning/dirDb", "partitioning/dirTable", "partitioning/dirColName", "partitioning/latColName",
-        "partitioning/lonColName", "partitioning/overlap", "partitioning/secIndexColName"};
+        "partitioning/lonColName", "partitioning/overlap", "partitioning/secIndexColName",
+        "partitioning/lockInMem", "partitioning/scanSpeed"};
```

Comment:
At the KV level I'd suggest to put those parameters into separate group, would simplify modification of those values if we need to do it later. Something like `"sharedScan/lockInMem", "sharedScan/scanSpeed"`.